src/platforms/dotnet/guides/winui/index.mdx

```diff
@@ -7,9 +7,9 @@ redirect_from:
 
 Sentry's .NET SDK works with Windows UI Library applications through the [Sentry NuGet package](https://www.nuget.org/packages/Sentry).
 
-## Configuration (for Just-In-Time compilation)
+## Configuration (without trimming)
```

Comment:
```suggestion
## Configuration (Without Trimming)
```

Reply:
These headers are actually supposed to be title case. Sorry for missing it last time!